Xamarin.Forms.Platform.WPF/Renderers/ProgressBarRenderer.cs

```diff
@@ -19,12 +13,13 @@ protected override void OnElementChanged(ElementChangedEventArgs<ProgressBar> e)
 			{
 				if (Control == null) // construct and SetNativeControl and suscribe control event
 				{
-					SetNativeControl(new WProgressBar { Minimum = 0, Maximum = 1, Foreground = Brushes.DeepSkyBlue });
+					SetNativeControl(new WProgressBar { Minimum = 0, Maximum = 1 });
 					Control.ValueChanged += HandleValueChanged;
 				}
 
 				// Update control property 
 				UpdateProgress();
+				UpdateProgressColor();
```

Comment:
In some renderers, `UpdateProgress()` is called before `UpdateProgressColor()` while in others, it's the reverse. I'd prefer consistency as long as it's okay to be consistent. Same with the way `OnElementPropertyChanged` is laid out.

Reply:
@adrianknight89 that's fine. order doesn't matters